routes/fileUpload.js

```diff
@@ -1,8 +1,9 @@
 const express = require("express");
 const FileController = require("../controllers/FileUploadController");
+const multer = require("multer");
 
 const router = express.Router();
 
-router.post("/", FileController.FileUploads);
+router.post("/", multer().single('file'), FileController.FileUploads);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7769089)
```suggestion
router.post("/", multer().single("file"), FileController.FileUploads);
```